react-native-libraries.json

```diff
@@ -11630,5 +11630,25 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/pmndrs/zustand",
+    "npmPkg": "zustand",
+    "examples": [
+      "https://zustand-demo.pmnd.rs/",
+      "https://github.com/pmndrs/zustand/tree/main/examples/demo"
+    ],
+    "images": [
+      "https://github.com/pmndrs/zustand/raw/main/docs/bear.jpg"
+    ],
+    "android": true,
+    "ios": true,
+    "web": true,
+    "expoGo": true,
+    "unmaintained": false,
+    "dev": false,
+    "template": false,
+    "newArchitecture": true,
+    "newArchitectureNote": "Zustand is a lightweight state management solution that works well with React Native's new architecture."
```

Comment:
```suggestion
```

This field is reserved for explaining a New Architecture status, if support state is more complicated than boolean value.